### PR TITLE
Using container component instead of div

### LINKED ISSUE #16: Use container element from neeto components

### PR DESCRIPTION
**Pull request by sritabh**
fixes #16 
- Fixed:
 -- Added Function to display time in particular format 
-- Variable name stayling fixed
-- Completed TODOs

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Item.jsx [hunk 1]
```diff
@@ -39,13 +41,11 @@ const Item = ({ note, showDeleteAlertForNote, showEditNotePane }) => {
...
         </div>
-        {/* TODO: Add function to calculate times_ago for tooltip */}
-        <Tooltip content="Wednesday, 10:30 AM" position="bottom-start">
+        <Tooltip content={formatDate(note.createdAt)} position="bottom-start">
           <div className="flex items-center space-x-2">
             <Clock size="15" />
             <Typography className="flex items-start " style="body1">
-              {/* TODO: Add function to calculate times_ago */}
-              {note.status} 2 hours ago
+              {note.status} {timesAgo(note.createdAt)}
```
Comment (navaneethsdk) [inline app/javascript/src/components/Dashboard/Notes/Item.jsx:48]: I feel that the name `timesAgo` does not properly convey the required functionality. Don't you think something like `findElapsedTime` would be more suitable here?

File: app/javascript/src/components/Dashboard/Notes/utils.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    minute: "numeric",
+    hour12: true,
+  };
+
+  if (date.toDateString() !== currentDateAndTime.toDateString()) {
+    options.day = "numeric";
+  }
+
+  return date.toLocaleString("en-US", options);
+};
```
Comment (navaneethsdk) [inline app/javascript/src/components/Dashboard/Notes/utils.js:35]: Please use[`Day.js`](https://day.js.org/docs/en/installation/installation). Refer: https://academy.bigbinary.com/learn-react/react-best-practices/miscellaneous-tips?searchTerms%5B0%5D=dayjs#prefer-dayjs-over-inbuilt-date-functions

**Review by navaneethsdk — COMMENTED** [review]
@sritabh A few comments

File: app/javascript/src/components/Dashboard/Notes/Item.jsx (hunk 1 above)
Comment (sritabh) [reply app/javascript/src/components/Dashboard/Notes/Item.jsx:48]: Yes, sounds much better.
Will do that.

**Pull request merged**